### DISCUSSION TITLE
Infinite redirection on authentication/token route

### DISCUSSION THREAD

**Discussion by AronWebSolutionPVTLTD** [Kyon147/laravel-shopify#339] [Q&A]
Hi, @Kyon147 

I have a installation (20.0), and I am facing an infinite loop on authenticate/token page.
I want it to redirect back to the home route with the host but it's showing the blank page in the embedded app iframe.

https://admin.shopify.com/store/m24logistics/apps/m24-logistics/authenticate/token?shop=m24logistics.myshopify.com&target=%2F%3Fhost%3DYWRtaW4uc2hvcGlmeS5jb20vc3RvcmUvbTI0bG9naXN0aWNz&host=YWRtaW4uc2hvcGlmeS5jb20vc3RvcmUvbTI0bG9naXN0aWNz

![image](https://github.com/user-attachments/assets/168c7756-9577-455f-abd8-c2adedac7dc5)

This issue only occurs the first time after the app is installed, where it fails to redirect to the home page. However, if I click on another page,redirection works fine, and even from that other page, the home page also redirects correctly. The problem only happens the first time after the app is installed.

Please help me to fix this.

**Comment by Kyon147** [comment]
This is a known issue on blade, but I can't replicate it easily but I think it is the redirect URL for the blade token route and if it passes itself, it just redirects indefinitely.
